test/test_bsonjs.py

```diff
@@ -63,6 +66,12 @@ def bsonjs_loads(json_str):
     return to_object(bsonjs.loads(json_str))
 
 
+BSONJS_JSON_OPTIONS = json_util.JSONOptions(
+                                  json_mode=json_util.JSONMode.CANONICAL,
+                                  uuid_representation=UuidRepresentation.PYTHON_LEGACY,
+                                  tz_aware=True)
```

Comment:
These three lines should only be indented 4 spaces.

Reply:
Done.